copydb/cmd/main.go

```diff
@@ -99,6 +78,26 @@ func main() {
 		errorAndExit(fmt.Sprintf("failed to parse config file: %v", err))
 	}
 
+	if stateFilePath != "" {
+		if _, err := os.Stat(stateFilePath); os.IsNotExist(err) {
+			errorAndExit(fmt.Sprintf("%s does not exist", stateFilePath))
+		}
+
+		f, err := os.Open(stateFilePath)
+		if err != nil {
+			errorAndExit(fmt.Sprintf("failed to open state file: %v", err))
+		}
+
+		resumeState := &ghostferry.SerializableState{}
+		parser := json.NewDecoder(f)
+		err = parser.Decode(&resumeState)
+		if err != nil {
+			errorAndExit(fmt.Sprintf("failed to parse state file: %v", err))
+		}
+
+		config.StateToResumeFrom = resumeState
```

Comment:
This will overwrite if the state to resume is specified twice. I suppose that's ok?

Reply:
yeah, state in filepath always wins.